src/Lykke.Service.SmsSender/Controllers/CallbackController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Cqrs;
+using Lykke.Service.SmsSender.Core.Domain.SmsRepository;
+using Lykke.Service.SmsSender.Models;
+using Lykke.Service.SmsSender.Sagas.Commands;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Service.SmsSender.Controllers
+{
+    [Route("callback")]
+    [ApiExplorerSettings(IgnoreApi=true)]
+    public class CallbackController : Controller
+    {
+        private readonly ISmsRepository _smsRepository;
+        private readonly ICqrsEngine _cqrsEngine;
+        private readonly ILog _log;
+
+        public CallbackController(
+            ISmsRepository smsRepository,
+            ICqrsEngine cqrsEngine,
+            ILog log
+        )
+        {
+            _smsRepository = smsRepository;
+            _cqrsEngine = cqrsEngine;
+            _log = log;
+        }
+
+        [HttpPost]
+        [Route("twilio")]
+        public async Task<IActionResult> TwilioCallback([FromForm] TwilioCallbackModel model)
+        {
+            if (model == null)
+                return Ok();
+            
+            _log.WriteInfo(nameof(TwilioCallback), model, "Twilio callback");
+            
```

Comment:
and will be good add in context CountryCode

Reply:
it's just logging of the callback request itself, the country code is logged in the SmsCommandHandler